src/azure-cli/azure/cli/command_modules/network/tests/latest/test_dns_commands.py

```diff
@@ -138,7 +138,7 @@ class DnsScenarioTest(ScenarioTest):
     @ResourceGroupPreparer(name_prefix='cli_test_dns')
     def test_dns(self, resource_group):
 
-        self.kwargs['zone'] = 'myzone.com'
+        self.kwargs['zone'] = 'myzonex.com'
```

Comment:
why `myzone` becomes `myzonex`

Reply:
It passes when changed, otherwise it will fail when I run live test

Reply:
May I ask is this because of the conflicts with existing resources? If so, can we change it to a random value to avoid the same problem in live runs in the future?